src/main/java/io/github/codeutilities/CodeUtilities.java

```diff
@@ -76,6 +77,7 @@ public void onInitialize() {
         loader.load(new AutomationFeature());
         loader.load(new Client());
         loader.load(new StreamerModeListeners());
+        new SocketHandler();
```

Comment:
should probably be loader.load, besides that code should be good even tho i didnt test it